src/pipert2/utils/method_data.py

```diff
@@ -8,5 +10,19 @@
 
 @dataclass
 class Method:
-    name: str
+    event_name: str
+    routines_by_flow: Dict[str, Optional[list]] = None
     params: dict = field(default_factory=lambda: {})
+
+    def is_flow_valid(self, flow_name_to_validate: str):
+        if self.routines_by_flow:
+            return flow_name_to_validate in self.routines_by_flow.keys()
+        else:
+            return True
+
+    def is_contain_routines(self, flow_name_to_validate: str):
+        if self.routines_by_flow:
+            return flow_name_to_validate in self.routines_by_flow.keys() and \
```

Comment:
You don't have to request the keys when checking if key exists in dict.

Reply:
I'm checking if the specific requested flow has routines.

Reply:
lets say we have the next dict:
`
x = {
    "flow1" : ["r1", "r2"]
}`

`flow_name_to_validate = "flow1"`

`flow_name_to_validate in x`
will be the same as
`flow_name_to_validate in x.keys()`

Or maybe i am not getting it right.